test/fixtures/v1-02807-state-items.json

```diff
@@ -22,41 +22,17 @@
   },
   "data_partners": {},
   "incentives": [
-    {
-      "payment_methods": [
-        "pos_rebate"
-      ],
-      "authority_type": "federal",
-      "program": "Federal Home Electrification and Appliance Rebates (HEAR)",
```

Comment:
This went away (along with the 25C credit below) because I removed `heat_pump_air_conditioner_heater` from the requested items.

The federal-level HEAR rebates are still being served from the API because they need to stay in v0 for the old calculator embeds still out there. I could remove them from v1, though; I can do that in another followup.